src/nn/graph.ts

```diff
@@ -151,9 +127,11 @@ export class MLGraph {
   private operandRefs_: Map<MLOperand, number> = new Map();
   private constantTensors_: Map<ConstantOperand, tf.Tensor> = new Map();
 
-  compute(inputs: MLNamedInputs, outputs: MLNamedOutputs): void {
+  /** @internal */
+  async compute(
+    inputs:MLNamedArrayBufferViews, outputs: MLNamedArrayBufferViews,
```

Comment:
nit: a blank between inputs: and MLNamedArraybufferViews

Reply:
Updated, thanks.